intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -515,6 +523,19 @@ \subsection{Check your quota}
 % Scratch        11  100000  150000       20   none
 % \end{prompt}
 
+\ifgent
+If you obtained your VSC account via \university, you can consult your current storage quota usage
```

Comment:
I would replace this sentence by just:
`You can consult your current \university storage quota...`  if we use `\ifgent` we assume that this documentation is for UGent users and you do not need to copy/paste the previous tip.

Reply:
@alvarosimon fixed in f24bbe9